File: iOS/sampleObjectC/sampleObjectC/MessageEventModel.m
//
//  MessageEventModel.m
//  sampleObjectC
//
//  Created by oshumini on 2017/6/16.
//  Copyright © 2017年 HXHG. All rights reserved.
//

#import "MessageEventModel.h"

@interface MessageEventModel ()
@end

@implementation MessageEventModel

- (instancetype)initWithMsgId:(NSString *)msgId eventText:(NSString *)eventText
{
  self = [super init];
  if (self) {
    _msgId = msgId;
    _evenText = eventText;
  }
  return self;
}


@end

File: iOS/sample/sample/MessageEventModel.swift
//
//  MessageEventModel.swift
//  sample
//
//  Created by oshumini on 2017/6/16.
//  Copyright © 2017年 HXHG. All rights reserved.
//

import Foundation

class MessageEventModel: NSObject, IMUIMessageProtocol {

  var msgId: String = ""
  var eventText: String = ""
  
  init(msgId: String, eventText: String) {
    super.init()
    self.msgId = msgId
    self.eventText = eventText
  }
}
